src/sdg/scss/components/_tabs.scss

```diff
@@ -100,6 +100,32 @@
     line-height: 1.5;
     color: #212121;
 
+    &--warning {
+      background: repeating-linear-gradient(
+        -45deg,
+        $color_accent_notification-bg,
+        $color_accent_notification-bg 5px,
+        darken($color_accent_notification-bg, 10%) 5px,
+        darken($color_accent_notification-bg, 10%) 10px
+      );
+      border: 1px solid darken($color_accent_notification-bg, 10%);
+      color: $color_accent_darkest;
+      height: 22px;
+      font-size: 10px;
+      padding: 4px 8px;
+      text-align: end;
+      position: relative;
+      top: -1px;
+      filter: opacity(0.8);
+      transition: all 0.3s ease-in-out;
```

Comment:
Try to keep your formatting in order, it's tyically better not to nest to deep and have newlines between selectors. Bonus points for sorting properties alphatibetically.